### PR TITLE
[com_privacy][3.9] Make urgent less meh, give more room for date

### LINKED ISSUE #22704: [com_privacy] Urgent is grey and "meh"
### Steps to reproduce the issue

Have an overdue request in com_privacy

### Expected result

[Urgent] label to instill a sense of urgency and be very evident and in your face

### Actual result

Grey washed out label

screenshot 2018-10-18 at 16 28 34

E.g on the dashboard it is!
screenshot 2018-10-18 at 16 31 12

### PR DESCRIPTION
**Pull request by PhilETaylor**
Pull Request for Issue #22704 .

### Summary of Changes

Have an overdue request in com_privacy
Urgent label now red - and better positioned to the right 
Date to have enough room to breath on wider range of monitor resolutions, was wrapping rendered date even on half a 27" iMac on 2560x1440!

### Expected result

[Urgent] label to instill a sense of urgency and be very evident and in your face
screenshot 2018-10-20 at 14 36 42

### previous look

Grey washed out label

screenshot 2018-10-18 at 16 28 34

### CODE REVIEW

File: administrator/components/com_privacy/views/requests/tmpl/default.php [hunk 1]
```diff
@@ -96,12 +96,12 @@
 								<?php echo JHtml::_('PrivacyHtml.helper.statusLabel', $item->status); ?>
 							</td>
 							<td>
+                                <?php if ($item->status == 1 && $urgentRequestDate >= $itemRequestedAt) : ?>
```
Comment (mbabker) [inline administrator/components/com_privacy/views/requests/tmpl/default.php]: Tabs...

Comment (PhilETaylor) [reply administrator/components/com_privacy/views/requests/tmpl/default.php]: I hate PHPStorm. And now my HDD has failed, again, I dont have this code locally :(

**Review by mbabker — APPROVED** [review]

File: administrator/components/com_privacy/views/requests/tmpl/default.php (hunk 1 above)
Comment (mbabker) [reply administrator/components/com_privacy/views/requests/tmpl/default.php]: Fixed.

Comment (PhilETaylor) [reply administrator/components/com_privacy/views/requests/tmpl/default.php]: Thanks...

**Comment by alikon** [comment]
I have tested this item :white_check_mark: successfully on 755ce80f0669fe335e5d95bfdfa77e24e7730224<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/22746">issues.joomla.org/tracker/joomla-cms/22746</a>.</sub>

**Comment by laoneo** [comment]
I have tested this item :white_check_mark: successfully on 755ce80f0669fe335e5d95bfdfa77e24e7730224<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/22746">issues.joomla.org/tracker/joomla-cms/22746</a>.</sub>

**Pull request merged**